core/cautils/sliceutils.go

```diff
@@ -0,0 +1,27 @@
+package cautils
+
+import "github.com/kubescape/opa-utils/reporthandling"
+
+func IsSubSliceScanningScopeType(haystack []reporthandling.ScanningScopeType, needle []reporthandling.ScanningScopeType) bool {
```

Comment:
I am pretty sure there are more easily readable algorithms that can achieve that